src/hooks/xcm/useTransferRouter.ts

```diff
@@ -298,7 +298,7 @@ export function useTransferRouter() {
     if (!isLocalTransfer.value) {
       // if: XCM bridge
       const selectedNetwork = xcmOpponentChain.value;
-      const isAstarEvm = from.value.includes(pathEvm) || to.value.includes(pathEvm);
+      const isAstarEvm = !!(from.value?.includes(pathEvm) || to.value?.includes(pathEvm));
```

Comment:
Let's use Boolean for it!
      const isAstarEvm = Boolean(from.value?.includes(pathEvm) || to.value?.includes(pathEvm));

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1294/commits/8319558d1c40548b3fc36f649cc7a5b1cb12bbc2) commit.